app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntrySheet.kt

```diff
@@ -108,12 +130,47 @@ class BloodPressureEntrySheet : BottomSheetActivity(), BloodPressureEntryUi {
             dayTextChanges(),
             monthTextChanges(),
             yearTextChanges()
-        ),
+        )
+        .compose(ReportAnalyticsEvents())
```

Comment:
I just realised that doing this is a little dangerous since `BloodPressureEntrySheetController` also forwards events to analytics. This means that until the controller is completely migrated and deleted, the events will be pushed twice to analytics. This is fine if we are sure that we'll finish the migration in time for the next release, but it's still something to watch out for.

Reply:
Good catch, I missed removing `ReportAnalyticsEvents()` from the controller. Which I can do now. Therefore we could continue shipping the feature even if we are in the middle of refactoring.

Reply:
Fixed

Reply:
Should we look at capturing the recorded analytics events as well in the characteristics tests, so we know that the events we record have not changed in any way?